eksctl/dandi.jsonnet

```diff
@@ -28,6 +28,15 @@ local notebookNodes = [
     { instanceType: "r5.xlarge" },
     { instanceType: "r5.4xlarge" },
     { instanceType: "r5.16xlarge" },
+    {
+     instanceType: "g4dn.xlarge",
```

Comment:
@yuvipanda should dandi have 2xlarge also like the linc hub?

The cost is ~50% higher (0.75 USD/hour compared to 0.53)

Reply:
Ahhttps://github.com/2i2c-org/infrastructure/issues/3824#issuecomment-2024299035 declared that to be the case.

Reply:
@consideRatio, I've added https://github.com/2i2c-org/infrastructure/pull/3866/commits/392979ce325791b8c90240d635b656f3c5897cd8 to also support 2xlarge gpu machines. Does that look right to you or should we have _only_ 2xlarge machines?

Reply:
@GeorgianaElena yep, we should have both.